packages/pluggable-widgets-tools/package.json

```diff
@@ -95,36 +108,33 @@
     "sass": "^1.43.4",
     "semver": "^7.3.2",
     "shelljs": "^0.8.4",
-    "ts-jest": "^26.4.3",
+    "ts-jest": "^29.0.0",
     "ts-node": "^9.0.0",
-    "typescript": "4.5.4",
+    "typescript": "^4.7.0",
     "xml2js": "^0.4.23",
     "zip-a-folder": "^0.0.12"
   },
   "devDependencies": {
     "@mendix/generator-widget": ">=8.9.0",
-    "@types/xml2js": "^0.4.5"
+    "@types/xml2js": "^0.4.5",
+    "async-mutex": "^0.2.4",
+    "shx": "^0.3.3",
+    "tree-kill": "^1.2.2",
+    "yeoman-test": "^6.2.0"
+  },
+  "scripts": {
+    "prepare": "shx rm -rf dist && tsc",
+    "test": "jest",
+    "test:scripts": "npm run prepare && node tests/commands.js"
   },
```

Comment:
The rebase for this file reverted some configurations.
The `test:scripts` have been moved to the root and the previews package.json (before this change) was organized in a way to avoid the necessity to scroll down to see important configurations like the available scripts

Reply:
Fixed